infra/tf/variables.tf

```diff
@@ -0,0 +1,106 @@
+################################################################################
+# ZQ2 GCP Terraform input variables
+################################################################################
+
+
+variable "project_id" {
+  description = "Project ID used to dpeloy the ZQ2 network"
+  type        = string
+  nullable    = false
+}
+
+variable "region" {
+  description = "The region ID to host the network in"
+  type        = string
+  default     = "europe-west2"
+}
+
+variable "node_zone" {
+  type     = string
+  default  = "europe-west2-a"
+  nullable = false
+}
+
+variable "eth_chain_id" {
+  description = "ZQ2 EVM Chain ID"
+  type        = number
+  nullable    = false
+  default     = 33469
```

Comment:
Likewise for this - This value only makes sense for the devnet, so I don't think it should be the default.